contact.html

```diff
@@ -82,7 +82,7 @@
               <li><a class="nav-link active" href="index.html">Home</a></li>
               <li><a class="nav-link" href="about.html">About</a></li>
               <li><a class="nav-link" href="courses.html">Courses</a></li>
-              <li><a class="nav-link" href="reaserch.html">Reaserch</a></li>
+              <li><a class="nav-link" href="research.html">Reaserch</a></li>

```

Comment:
`Reaserch` should be corrected to `Research`.

Reply:
yes , sorry for silly mistake